server/routerlicious/packages/services-core/src/index.ts

```diff
@@ -11,6 +11,7 @@ export * from "./combinedLambda";
 export * from "./combinedProducer";
 export * from "./database";
 export * from "./document";
+export * from "./emptyTaskMessageSender";
```

Comment:
Curious what this file is for and why we need to add these exports here?

Reply:
This is the index.js file which is an entrypoint. Adding exports here let us do things like import { Something } from "packagename"